app/models/event.rb

```diff
@@ -32,18 +32,20 @@ class Event < ApplicationRecord
     :hide_title,  # bool
     :organizer_email, # string
     :receipt_message, # text
-    :nonprofit
+    :nonprofit,
+    :in_person_or_virtual
+
+  enum :in_person_or_virtual, {in_person: "in_person", virtual: "virtual"}, validate: true
```

Comment:
You can avoid writing out the key and value by using `itself`
```suggestion
  enum :in_person_or_virtual, %w[in_person virtual].index_by(&:itself), validate: true
```

Reply:
Good thinking, let's do that!